packages/eslint-plugin-pf-codemods/lib/rules/v5/emptyStateIcon-icon-required.js

```diff
@@ -0,0 +1,30 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8737
+module.exports = {
+  meta: {},
+  create: function (context) {
+    const imports = getPackageImports(context, "@patternfly/react-core").filter(
+      (specifier) => "EmptyStateIcon" === specifier.imported.name
+    );
+    return imports.length === 0
+      ? {}
+      : {
+          JSXOpeningElement(node) {
+            if (imports.find((imp) => imp.local.name === node.name.name)) {
```

Comment:
If we implemented the above comment, we also could avoid having to `.map` here and just check if the import `local.name === node.name.name`